xmpp/xmpp.go

```diff
@@ -51,6 +51,10 @@ type item struct {
 	Topic           string `xml:"x>topic"`
 }
 
+type Ack struct {
+	Ack string `xml:"a"`
+}
+
```

Comment:
@lusis `Ack` was added, but not used? is your intention for the library to leverage this or the dev?